src/doc/en/installation/meson.rst

```diff
@@ -135,21 +136,23 @@ Depending on your distribution, install the following packages:
     At least Fedora 41 is required.
 
    .. literalinclude:: fedora.txt
-
+      :language: console
```

Comment:
```suggestion
      :language: bash
```
If in addition to that the $ is suppressed from fedora.txt one can copy the contents to the terminal.

Reply:
So you mean, don't write the `$` in fedora.txt?

That might work indeed. Maybe as a follow-up PR?

Reply:
OK. I will try one.